PerimeterXModule/Internals/Validators/PXS2SValidator.cs

```diff
@@ -149,9 +149,27 @@ public RiskResponse SendRiskResponse(PxContext PxContext)
 				riskRequest.Additional.OriginalTokenError = PxContext.OriginalTokenError;
 			}
 
-			string requestJson = JSON.SerializeDynamic(riskRequest, PxConstants.JSON_OPTIONS);
+			SetCredentialsIntelligenceOnRisk(PxContext, riskRequest.Additional);
+
+            string requestJson = JSON.SerializeDynamic(riskRequest, PxConstants.JSON_OPTIONS);
 			var responseJson = httpHandler.Post(requestJson, PxConstants.RISK_API_PATH);
 			return JSON.Deserialize<RiskResponse>(responseJson, PxConstants.JSON_OPTIONS);
 		}
+
+		public void SetCredentialsIntelligenceOnRisk(PxContext pxContext, Additional riskRequest)
+		{
+			LoginCredentialsFields loginCredentialsFields = pxContext.LoginCredentialsFields;
+
+            if (loginCredentialsFields != null)
+			{
+                riskRequest.Username = loginCredentialsFields.Username;
```

Comment:
fix indentation in file

Reply:
Everything looks fine in the source file (and not on Github)